src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsComboBoxUI.java

```diff
@@ -517,34 +517,6 @@ WindowsComboBoxUI getWindowsComboBoxUI() {
         }
     }
 
-
-    /**
-     * Subclassed to add Windows specific Key Bindings.
-     * This class is now obsolete and doesn't do anything.
-     * Only included for backwards API compatibility.
-     * Do not call or override.
-     *
-     * @deprecated As of Java 2 platform v1.4.
-     */
-    @Deprecated
```

Comment:
Should we not mark it "forRemoval=true" before actually removing it?

Reply:
it is not needed, this is not a public API.